### DISCUSSION TITLE
Members cannot invite outside collaborators

### DISCUSSION THREAD

**Discussion by dufagundes** [community/community#25185] [Actions]
<p>I am admin on the repo, I am able to search outside collaborators, but when click t send the invite, I am getting this message. Not sure what my company admin need to change on my profile. </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/dee482edf0a44190c322e645331a3c4f0ccf34f0.png" data-download-href="/uploads/short-url/vNNA50GpKtYYOcmJi74yCloj9gA.png?dl=1" title="GitHub.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/15809257/181089295-9e125a0d-891f-43b3-aad7-de256175289e.png" alt="GitHub.png" data-base62-sha1="vNNA50GpKtYYOcmJi74yCloj9gA" width="690" height="450" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/dee482edf0a44190c322e645331a3c4f0ccf34f0_2_690x450.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/dee482edf0a44190c322e645331a3c4f0ccf34f0_2_1035x675.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/dee482edf0a44190c322e645331a3c4f0ccf34f0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/dee482edf0a44190c322e645331a3c4f0ccf34f0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">GitHub.png</span><span class="informations">1192×779 56 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/dufagundes">@dufagundes</a> ,</p>
<p>According to the description from the docs about <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/adding-outside-collaborators-to-repositories-in-your-organization" rel="nofollow noopener"><strong>Adding outside collaborators to repositories in your organization</strong></a>:</p>
<blockquote>
<p>Only members with owner privileges for an organization or admin privileges for a repository can add outside collaborators to a repository, unless an organization owner has restricted the ability to invite collaborators.</p>
</blockquote>
<p>So you can go to check if organization owners have set restriction for adding outside collaborators, more details, you can reference the docs about <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/setting-permissions-for-adding-outside-collaborators" rel="nofollow noopener"><strong>Setting permissions for adding outside collaborators</strong></a>. If so, you need to contact the organization owners to ask for adding the outside collaborators or disabling the restriction.</p>
